### PR TITLE
Rcal-119: dq docs

### LINKED ISSUE #159: DQ update to read the docs
_Issue [RCAL-119](https://jira.stsci.edu/browse/RCAL-119) was created on JIRA by [David Davis](https://jira.stsci.edu/secure/ViewProfile.jspa?name=ddavis):_

Update read the docs with the DQ reference file information using <https://jwst-pipeline.readthedocs.io/en/latest/jwst/references_general/references_general.html#reference-file-types> as a starting point.

### PR DESCRIPTION
**Pull request by PaulHuwe**
Closes #159 

Resolves RCAL-119

**Description**

This PR addresses ...
This PR updates the RTD files to include dq_init.

Checklist
- [ ] Tests

- [ x ] Documentation

- [ ] Change log

- [ ] Milestone

- [ ] Label(s)

### CODE REVIEW

**Comment by PaulHuwe** [comment]
The force push was a rebase to remove the DQ_Init code commits from PR-262.

File: docs/roman/package_index.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@ Package Index
 
    datamodels/index.rst
    dark_current/index.rst
+   dq_init//index.rst
```
Comment (dmggh) [inline docs/roman/package_index.rst:9]: 2nd '/' will be ignored

File: docs/roman/dq_init/description.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Description
+============
+The Data Quality (DQ) initialization step in the calibration pipeline
+populates the DQ mask for the input dataset. Flag values from the
+appropriate static mask ("MASK") reference file in CRDS are copied into the
+"PIXELDQ" array of the input dataset, because it is assumed that flags in the
+mask reference file pertain to problem conditions that affect all groups and
```
Comment (dmggh) [inline docs/roman/dq_init/description.rst:7]: Not that anyone would get confused, 'all ... integrations' is for the single integration of course.

**Review by dmggh — APPROVED** [review]
2nd '/' ignored

File: docs/roman/dq_init/description.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+   through pipeline. This will create "pixeldq" and "groupdq" arrays (if they 
+   don't already exist).
+
+ - Propagate the DQ flags from the reference file DQ array to the science data "PIXELDQ"
+   array using numpy's ``bitwise_or`` function.
+
+Note that when applying the ``dq_init`` step to guide star data, as is done in
+the TBD
+#:ref:`calwebb_guider <calwebb_guider>` 
+pipeline, the flags from the MASK reference
```
Comment (ddavis-stsci) [inline docs/roman/dq_init/description.rst:25]: You can remove the calwebb_guider part. Right now there are no plans for a Roman equivalent.

**Review by ddavis-stsci — APPROVED** [review]

File: docs/roman/package_index.rst (hunk 1 above)
Comment (PaulHuwe) [reply docs/roman/package_index.rst:9]: Removed.

File: docs/roman/dq_init/description.rst (hunk 2 above)
Comment (PaulHuwe) [reply docs/roman/dq_init/description.rst:7]: Removed the integrations.

File: docs/roman/dq_init/description.rst (hunk 3 above)
Comment (PaulHuwe) [reply docs/roman/dq_init/description.rst:25]: Removed.

**Pull request merged**